components/PageNotFound.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
+    <v-container>
+      <v-stack
+        justify="space-between"
+        md-direction="column-reverse"
+        align="center"
+        md-justify="space-evenly"
+        class="error-content-container"
+      >
+        <v-image
+          path="images/404-illustration.png"
+          class="error-illustration"
+        />
+        <v-stack direction="column" class="error-content">
+          <v-heading>Page Not Found!</v-heading>
+        </v-stack>
+      </v-stack>
+    </v-container>
+  </section>
+</template>
+
+<style lang="postcss" scoped>
+.error-content-container {
+  height: 80vh;
+  display: flex;
+  flex-direction: column;
```

Comment:
The `display: flex` property is set by the `v-stack` implicitly and we can set `flex-direction: column` with a `direction="column"` prop. 

Check if this works. The `md-direction` and `md-justify` props are not required.

```
<v-stack direction="column" align="center" justify="space-between" class="...">
  ...
</v-stack>
```
```

Reply:
done [commit](https://github.com/arcana-network/website/pull/85/commits/49160c8a55fa3b8aeb21061e4e50cb6e2a66f1ef)

Reply:
Can we also remove `md-direction` and `md-justify`?

Reply:
sorry, missed that. done. [commit link](https://github.com/arcana-network/website/pull/85/commits/bd9447f16b7bd6955ba37d3d8817aabf29159819)

Reply:
this [commit](https://github.com/arcana-network/website/pull/85/commits/c0f7fce3a07e6d7479ee90f190b3bf0fc7e79dd7) is needed to align element on mobile screens

### screenshot:
<img width="331" alt="Screenshot 2022-02-09 at 1 06 24 PM" src="https://user-images.githubusercontent.com/40512676/153143474-10f4d9a6-c651-4a64-921c-8c9b4bf5e4c7.png">